source/run-command/tests/unified/runCommand.yml

```diff
@@ -65,6 +65,7 @@ tests:
                 $db: *db
                 lsid: { $$exists: true }
                 $readPreference: { $$exists: false }
+                $clusterTime: { $$exists: true }
```

Comment:
should the title be updated accordingly?

Reply:
I broke this out to its own test case, I believe I need to run a command to first get a gossiped clusterTime and it will be present on any subsequent command, checking in our driver CI.